### PR TITLE
MRC-42 CT Liability in results page

### PR DESCRIPTION
**Pull request by girishkamat**

### CODE REVIEW

File: test/connectors/sharedmodel/CalculatorResultSpec.scala [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  private val flatRate = FlatRate(1, 11, 111, 1111, 1111)
+
+  "FlatRate" - {
+    "marginalRelief" - {
+      "should return 0" in {
+        flatRate.marginalRelief shouldBe 0
+      }
+      "should return corporation tax" in {
+        flatRate.corporationTaxBeforeMR shouldBe 11
```
Comment (PavelVjalicin) [inline test/connectors/sharedmodel/CalculatorResultSpec.scala:32]: (Optional) I would change this line to use the flatRate value instead:
```scala
flatRate.corporationTaxBeforeMR shouldBe flatRate.corporationTax
```
So we would not need to modify the test if we need to change the flatRate values

File: test/connectors/sharedmodel/CalculatorResultSpec.scala [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      "sh"
+    }
+  }
+
+  private val marginalRate = MarginalRate(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11)
+
+  "MarginalRate" - {
+    "adjustedAugmentedProfit" - {
+      "should return sum of adjustedProfit and adjustedDistributions" in {
+        marginalRate.adjustedAugmentedProfit shouldBe 15
```
Comment (PavelVjalicin) [inline test/connectors/sharedmodel/CalculatorResultSpec.scala:45]: (Optional) Would change to (same as above):
```scala
marginalRate.adjustedAugmentedProfit shouldBe (marginalRate.adjustedProfit + marginalRate.adjustedDistributions)
```

**Review by PavelVjalicin — APPROVED** [review]
Looks good to me.

**Comment by girishkamat** [comment]
Also, merged with main and added rounding for all computed values in CalculatorResult

**Review by PavelVjalicin — DISMISSED** [review]

**Review by PavelVjalicin — APPROVED** [review]
Looks good to me.

**Pull request merged**